lib/src/util/number.dart

```diff
@@ -118,3 +119,46 @@ double moduloLikeSass(double num1, double num2) {
   var result = num1 % num2;
   return result == 0 ? 0 : result + num2;
 }
+
+/// Return square root of [number]
+SassNumber sqrt(SassNumber number) {
+  number.assertNoUnits();
+  return SassNumber(math.sqrt(number.value));
+}
+
+/// Return [num1]^[num2]
+SassNumber pow(SassNumber num1, SassNumber num2) {
+  num1.assertNoUnits();
+  num2.assertNoUnits();
+  return SassNumber(math.pow(num1.value, num2.value));
+}
+
+/// Return the rounded number according to the strategy
+SassNumber roundStrategies(SassString strategy, SassNumber number) {
```

Comment:
This shouldn't actually be necessary, since the CSS round requires both `number` and `step` to be provided, and we don't need to support different strategies for the legacy Sass round, so we can just use Dart's round instead.

Reply:
Done :)